README.md

```diff
@@ -1,3 +1,92 @@
+
+The goal of my analysis is to report on deep source activity (i.e. striatum in this case) in the task’s MEG data using synthetic aperture magnetometry (SAM) for source localization. If we find activity as expected in the high gamma band during the feedback phase of the task, it will become a useful way to study reward processing in mood disorders and the impact of novel treatments using MEG.
+
+Additionally, we plan to explore connectivity between regions related to reward processing using modelling (of some sort?), though the details of that are still being worked out.
+
+In terms of our analysis of behavior data, we need to gather mean reaction times and accuracy of button presses for analysis with an ANOVA (more details to come based on what factors will be involved).
+
+## Files that need to be processed:  
+-	Behavior data: a .txt output from the MID task that indicates button presses, timing of stimuli, etc.
+-	MEG data: a CTF (this is the MEG system we have at the NIH) file of MEG data from the task
+-	MRI data: structural MRIs for use with source localization (I already have a script for converting the files from the DICOM site to .nii files from my lab)
+
+# Processing steps:  
+I am not sure that is the best way to do this, but this is what needs to happen to get the data from its raw state to stats that I can report. The bolded script names are the ones I would like to create for my package based off a naming convention I’ve seen in our fMRI processing pipeline, though they are still subject to change:
+
+## Manual steps before scripts:
+1.	Transfer MEG, behavior, and MRI data to data processing folder on biowulf (this is just done with a BASH command)
+2.	Update demographics spreadsheet for use in analysis
+3.	Set fiducial markers on MRI copy
+
+##	MEG preprocessing step 1
+1.	Script name will be **01_megpreprocess1**, and I would like it to do the following:
+-	 Change file name using CTF command (to make file names look like “subject#_MID.ds” so they are more uniform)
+-	Set markers for stimuli (cue markers) and output file (.txt file) of the times of the cue marks (this file will be used in the next step)
+-	I already have some of the CTF commands for this, but I would like to figure out how to give this script (and all of the following ones) a list of subject names to loop through so I can do this all at once.
+2.	Input: MEG data file
+3.	Output: copied and edited MEG data file, cue markers text file
+
+##	Behavior processing
+1.	Script name will be **02_behprocess**, and it will do the following:
+-	Makes marker files for more MEG processing; these markers will designate win/loss/neutral cues to be marked in the MEG file and are output as three separate .txt files to the subject’s MEG directory
+-	Pulls columns from behavior data file and calculates mean RTs and accuracy by trial and subject and appends that to master behavior data sheet (I will probably work with this sheet in R unless I can do an ANOVA as another step)
```

Comment:
Look into pandas for working on tabular data. It will make things a lot easier. And using csv files instead of text might be useful.

Reply:
Python has advance tools for statistical learning/analysis. Statsmodels is probably what you need here but as the sophistication of your analysis grows you could add in the packages lme, scikit-learn, and tensorflow. There are certainly times to use R but if you can complete everything with python it will help you a lot. Learning how to debug bash, python, and R is never a fun task.